addon/templates/components/polaris-progress-bar.hbs

```diff
@@ -1,10 +1,12 @@
-<progress class="Polaris-ProgressBar__Progress"
+<progress
+  class="Polaris-ProgressBar__Progress"
   max="100"
-  value="{{parsedProgress}}"
+  value={{parsedProgress}}
```

Comment:
Test this very carefully, because this is probably now passing a number instead of a string which might break things.